### DISCUSSION TITLE
adding github as source to winget i get error

### DISCUSSION THREAD

**Discussion by soheilamiri** [microsoft/winget-pkgs#153828] [Q&A]
hi, 
i get error while attempting to add github community repository to winget

<details><summary>Details</summary>
<p>
winget source add --name GitHubCommunityRepo --arg https://github.com/microsoft/winget-pkgs
Adding source:
  GitHubCommunityRepo -> https://github.com/microsoft/winget-pkgs
An unexpected error occurred while executing the command:
0x80190194 : Not found (404).

</p>
</details>

**Comment by Trenly** [comment]
A GitHub repository can't be used directly, as WinGet requires either a Pre-Indexed package or a REST API as the source.

The community repository here is included with WinGet by default under the `winget` source - if you've removed it you can get it back by using `winget source reset --force` in an administrator terminal. Or, you can use `winget source add -n GitHubCommunityRepo -a https://cdn.winget.microsoft.com/cache -t Microsoft.PreIndexed.Package`
